var/spack/repos/builtin/packages/chai/package.py

```diff
@@ -39,12 +42,15 @@ class Chai(CMakePackage, CudaPackage, ROCmPackage):
     depends_on('cmake@3.8:', type='build')
     depends_on('cmake@3.9:', type='build', when="+cuda")
 
-    depends_on('blt@0.4.0:', type='build', when='@2.3.1:')
+    depends_on('blt@0.4.1', type='build', when='@2.4.0:')
     depends_on('blt@:0.3.6', type='build', when='@:2.3.0')
 
     depends_on('umpire')
     depends_on('raja', when="+raja")
 
+    depends_on('umpire@6.0.0', when="@2.4.0")
+    depends_on('raja@0.14.0', when="@2.4.0")
+
```

Comment:
Wouldn’t it be more accurate to say:
```suggestion
    depends_on('umpire@6.0.0:', when="@2.4.0")
    depends_on('raja@0.14.0:', when="@2.4.0")

```

Reply:
I _think_ that would typically allow to build with `@main` and `@develop`

Reply:
I don't want that to happen :D I I think that I could add:
`depends_on('umpire@6.0.0:', when="@develop"`
but I don't want people trying to build this CHAI version with RAJA/Umpire develop.

Reply:
Oh! Yes I see it now.
I guess we need to at least specify what `chai@main` and `chai@develop` depend on.